app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -1354,7 +1355,7 @@ protected void onPreExecute() {
         }
 
 
-        if (Authentication.isLoggedIn && NetworkUtil.isConnected(MainActivity.this))
+        if (Authentication.isLoggedIn && NetworkUtil.isConnected(MainActivity.this) && !FDroid.isFDroid)
```

Comment:
Just stick the `FDroid` check first so we don't hit `isConnected` when we don't need to.